### ISSUE TITLE
Make usable in Browser

### ISSUE THREAD

**Issue by skch-17** [NikolaiMe/textbox-for-pdfkit#1]
Hi first of all, nice library.

I needed exactly this layouting for "richtext" because the pdf kit does not function with diffrent font sizes at all. Or yea you have to write the callculation yourself, as you did.

So my request would be to make this library browser compatible. The blocking part is fontkit which is littarly the only dependency. I am still a beginner, so not sure if that's correct. So is there a possibility to get arround? Maybe add browserify to the build process?

Long story short: Add browser support.

**Comment by NikolaiMe** [comment]
Hi @skch-17,

Thank you for your input. I created this package for a backend project, I'm currently working on. As I'm currently just one person, who is mainly working on another project, I think I can not implement your request very soon. But if you like you can start contributing to the project by sending some pull requests. It seems as you would have already some good ideas how to solve this issue.

Regarding fontkit: This package is really necessary for that library, as it helps to measure the real font widths, so we can not just remove it. Due to their README they are supported in browser. As already said, I just used this package in backend until now.

If you like you can try using Browserify for this package. If it's working send a pull request. I'm very grateful for every support.

**Comment by skch-17** [comment]
Sorry for the late answer.

Currently I just set the "new" fonts to a default value, so that I could drop fontkit. I'am as well currently working on another project, so my time is a bit short as well. As soon I have time i will take a look at browsify.
